src/main/java/seedu/address/model/book/Rate.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.model.book;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a book's rate.
+ * Guarantees: immutable.
+ */
+public class Rate {
+
+    public final String rate;
+
+    /**
+     * Constructs a {@code Description}.
+     *
+     * @param rate A book rate.
+     */
+    public Rate(String rate) {
```

Comment:
This should take in an `int` `rating`.
Check the validity of the input using `AppUtil.checkArgument`, in a similar way to [this](https://github.com/nus-cs2103-AY1718S2/addressbook-level4/blob/master/src/main/java/seedu/address/model/person/Name.java).